librosa/core/audio.py

```diff
@@ -203,7 +203,7 @@ def to_mono(y):
 
 
 @cache
-def resample(y, orig_sr, target_sr, res_type='sinc_best', fix=True, **kwargs):
+def resample(y, orig_sr, target_sr, res_type='sinc_best', fix=True, scale=False, **kwargs):
```

Comment:
quick double check: does `scale=False` make sense here? I expected `True`, but maybe that'd be sane for `rescale`... just a thought